### ISSUE TITLE
Keyboard shortcuts

### ISSUE THREAD

**Issue by jtagcat** [prologic/todo#19]
Global shortcuts from https://github.com/openmicroapps/meta/issues/1

**Comment by jtagcat** [comment]
To show kb shortcuts, add a keyboard icon, what shows all shortcuts in their place. Either by replacing all text in place (so the + button becomes Enter) or adding a label to everything for say 8 seconds.
Oh wait, just have an enter icon there in the first place.. Bad example.

**Comment by prologic** [comment]
I think an icon makes sense (_somewhere_); but I also prefer the behavior you see in a lot of modern web apps.

- Press `?` to bring up keyboard shortcuts help
- It is displayed as an modal dialog / overlay thingie

**Comment by jtagcat** [comment]
When using an overlay, you need to explain everything. With the minimalist nature of no help pages, I like the show what buttons to press, where they are needed more.

**Comment by prologic** [comment]
I see; that could work too :)  Keeping to the minimal design

**Comment by jtagcat** [comment]
I am saying that we need a meta thread & rules somewhere.

**Comment by prologic** [comment]
I agree; but do you think it makes sense to quietly develop more of the ideas I had in mind before opening up an umbrella org and making a bigger deal of the larger effort? :)

**Comment by prologic** [comment]
Or !@#$ it :) Open Micro Apps actually sounds nice!

I can see the logo no :)

Open uApps

**Comment by jtagcat** [comment]
We can change the name and create the logo later, ? is uApps?
Let's try to move meta things off todo issue tracker and actually to meta.

**Comment by prologic** [comment]
agreed

**Comment by jtagcat** [comment]
We also would probably need a way to chat, I'm currently moving to Modular.im

**Comment by prologic** [comment]
I still use and prefer IRC :) [eris](https://github.com/prologic/eric) is technically also a self-hosted IRC server that I also had plans to use as the "chat" backend to a "chat" micro-app :)

**Comment by jtagcat** [comment]
@prologic you can use the irc bridge, there's even https://github.com/matrix-org/matrix-ircd
Having a Matrix based room has it's advantages, mainly it's easier to use and it has history.

**Comment by prologic** [comment]
Oooh very nice! cc @KevinNThomas in case this can be integrated with your outstanding and awesome PRs too :)
